x-pack/plugins/canvas/canvas_plugin_src/elements/index.ts

```diff
@@ -66,11 +66,16 @@ const elementSpecs = [
 
 const initializeElementFactories = [metricElementInitializer];
 
+const excludeElementsForDisabledPlugins = (element: ElementFactory, functions: string[]) => {
+  return !element().external || functions.find((fn) => element().expression_name === fn);
```

Comment:
nit: if you anyway created a separate method for that maybe it make sense to execute element factory once. 

```
const el = element(); 
return !el.external || functions.find((fn) => el.expression_name === fn);
```

Reply:
+ rename `functions.find -> functions.some`  just for semantic

Reply:
Done.